http/oidc/src/main/java/org/wildfly/security/http/oidc/LogoutHandler.java

```diff
@@ -151,7 +151,7 @@ private void handleBackChannelLogoutRequest(OidcHttpFacade facade) {
         String logoutToken = facade.getRequest().getFirstParam(LOGOUT_TOKEN_PARAM);
         TokenValidator tokenValidator = TokenValidator.builder(facade.getOidcClientConfiguration())
                 .setSkipExpirationValidator()
-                .setTokenType(LOGOUT_TOKEN_TYPE)
+                .setTokenType(LOGOUT_JWT_TOKEN_TYPE)
```

Comment:
With this change, won't logout fail when using Keycloak? We need a solution that will work for both Keycloak and other OpenID providers.

Reply:
I've tested this with the wfly integration tests and it does not fail.  Keycloak must be supporting both.

Reply:
Please debug to see what Keycloak's logout token looks like and what its claims are. It seems very strange that validation would pass before with checking the typ claim is set to "Logout" and now validation still passes with the typ claim set to "logout+jwt" instead. Is the validator actually getting invoked at all?

We also need to be careful because the spec says the typ value should be "logout+jwt", it doesn't say it MUST be set to this value.

Reply:
There is no way to determine which OpenID Provider is being used and thus
which claim 'typ' to use.  I have added code to use keycloak's 'typ' first.  
If that throws an exception the typ 'logout+jwt' is tried.

Reply:
Were you able to determine why the tests happened to be passing with Keycloak when attempting to validate that typ was set to 'logout+jwt'?

Reply:
I believe that code was not being called because isLogoutCallbackPath was not performing the proper check.